constants/projects.js

```diff
@@ -0,0 +1,20 @@
+export const BANNER_SIZES = ['mobile', 'touch', 'tablet', 'tablet-large', 'desktop'];
+
+export const PROJECTS = {
+  mapa: {
+    externalLink: 'https://map.wir.by?utm_source=wirby-main-page',
+    labelKey: 'mapa-title',
+    banners: {
+      desktop:
+        'https://res.cloudinary.com/wir-by/image/upload/c_scale,w_1020,f_auto,q_auto/v1568474406/production/banners/mapa-all-sizes/desktop.png',
```

Comment:
I think you can replace it with:
```
https://res.cloudinary.com/wir-by/image/upload/c_scale,w_${WIDTH_BY_SIZE[size]},f_auto,q_auto/v1568474405/production/banners/mapa-all-sizes/${size}.png
```

```js
const WIDTH_BY_SIZE = {
	desktop: 1020
}
```

Reply:
Well for the URLs I have right now - yes, I could've replaced it. But once it gets more complicated (also `v1568...` substring changes with time) it will require to rewrite code every time. Not cool.